catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/model/ExportCategory.java

```diff
@@ -138,9 +138,12 @@ private ExportValue simpleValue(Attribute attribute) {
   }
 
   private static boolean isNonEmptyValue(Metacard metacard, String attrName) {
-    final AttributeDescriptor descriptor =
-        metacard.getMetacardType().getAttributeDescriptor(attrName);
+    AttributeDescriptor descriptor = metacard.getMetacardType().getAttributeDescriptor(attrName);
     final Attribute attribute = metacard.getAttribute(attrName);
+    if (descriptor == null) {
+      return attribute != null && StringUtils.isNotEmpty(attribute.getValue().toString());
```

Comment:
We also need to check that `attribute.getValue()` is not null.

Reply:
I will add the check.